Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -609,6 +609,14 @@ function Accumulator_GetSpriteName(furniture)
 	return baseName .. "_" .. suffix
 end
 
+function Door_GetSpriteName(furniture)
+	if (furniture.verticalDoor) then
+	    return furniture.Type .. " Vertical_0"
+	else
+	    return furniture.Type .. "Horizontal_0"
+	end
+end
+
```

Comment:
Fixed (This also explains a bug that I rarely, but occasionally, half-saw that was driving me crazy)